pkg/sync/rpc/server.go

```diff
@@ -329,8 +330,12 @@ func (*SyncAgentServer) FileRename(ctx context.Context, req *ptypes.FileRenameRe
 
 func (s *SyncAgentServer) FileSend(ctx context.Context, req *ptypes.FileSendRequest) (*empty.Empty, error) {
 	address := net.JoinHostPort(req.Host, strconv.Itoa(int(req.Port)))
+	directIO := true
+	if filepath.Ext(strings.TrimSpace(req.FromFileName)) == ".meta" {
+		directIO = false
+	}
 	logrus.Infof("Sending file %v to %v", req.FromFileName, address)
-	if err := sparse.SyncFile(req.FromFileName, address, FileSyncTimeout); err != nil {
+	if err := sparse.SyncFile(req.FromFileName, address, FileSyncTimeout, directIO); err != nil {
```

Comment:
This seems like a change in functionality, not just adding logs.
Also log whether direct io is being used as part of the log call.

Reply:
seems like conflict with other issue https://github.com/longhorn/longhorn-engine/pull/560

Reply:
I probably misunderstood, but since your log changes are unrelated to the direct io changes in (#560) don't include them as part of this pr. Then we can merge this and sparse-files will be updated as part of the PR that introduces the direct io change?